includes/Story_Post_Type.php

```diff
@@ -556,4 +565,17 @@ public function filter_site_kit_gtag_opt( $gtag_opt ) {
 
 		return $gtag_opt;
 	}
+
+	/**
+	 * Add CPT to Jetpack sitemaps
+	 *
+	 * @param array $post_types Array of post types.
+	 *
+	 * @return array
+	 */
+	function add_to_sitemap( array $post_types ) {
```

Comment:
Let's call this `add_to_jetpack_sitemap`

Reply:
No, it is just just jetpack sitemaps. See https://github.com/google/web-stories-wp/pull/3264/files#diff-99d793dc332da60eaf6b6846bfd3a7f0R152

Reply:
I know, but `add_to_sitemap` is just too ambiguous, and `add_to_jetpack_or_dotcom_sitemap` is clearly not ideal either.

Reply:
Changed

Reply:
I don't think `add_to_jetpack_or_dotcom_sitemap` is ideal though.

I'd say `add_to_jetpack_sitemap` + clear description in the docblock is fine.